app/controllers/v0/sessions_controller.rb

```diff
@@ -33,6 +33,9 @@ def saml_callback
       if @saml_response.is_valid? && persist_session_and_user
         async_create_evss_account(@current_user)
         redirect_to SAML_CONFIG['relay'] + '?token=' + @session.token
+
+        obscure_token = Session.obscure_token(@session.token)
+        Rails.logger.info("Logged in user with id #{@session.uuid}, token #{obscure_token}")
```

Comment:
Oh, I _always_ have an opinion. :)

It's basically what you said -- we have to maintain a connection to the actual user to achieve the goals of the logging in the first place.  A UUID is considered linkable PII and not explicit PII and is allowable for this purpose.  If it makes you feel better to encrypt and hash the UUID and it doesn't introduce significant overhead, I'm cool with it.  I don't think just hashing it would really stop anyone who already had nefarious plans.